src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Random random = new Random();
+        Colors[] colors = Colors.values();
+        return colors[random.nextInt(colors.length)].toString();

```

Comment:
Don't use `toString()` or `String.valueOf()`(it will call `toString()` under the hood) for getting the `String` representation of enum constants. Use `name()` method instead.